apps/odyssey-stats/src/widget/modules.jsx

```diff
@@ -2,41 +2,118 @@ import { ShortenedNumber } from '@automattic/components';
 import { protect, akismet } from '@automattic/components/src/icons';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import wpcom from 'calypso/lib/wp';
 import useModuleDataQuery from '../hooks/use-module-data-query';
+import canCurrentUser from '../lib/can-current-user';
 
 import './modules.scss';
 
-function ModuleCard( { icon, title, value, isLoading, className = null } ) {
+function ModuleCard( {
+	icon,
+	title,
+	value,
+	activateProduct,
+	active,
+	className = null,
+	info = null,
+} ) {
+	const translate = useTranslate();
+	const [ disabled, setDisabled ] = useState( false );
 	return (
 		<div className={ classNames( 'stats-widget-module stats-widget-card', className ) }>
 			<div className="stats-widget-module__icon">{ icon }</div>
 			<div className="stats-widget-module__title">{ title }</div>
-			<div className="stats-widget-module__value">
-				{ isLoading ? '-' : <ShortenedNumber value={ value } /> }
-			</div>
+			{ ( active || ! activateProduct ) && (
+				<div className="stats-widget-module__value">
+					<ShortenedNumber value={ value } />
+				</div>
+			) }
+			{ ! active && activateProduct && (
+				<div className="stats-widget-module__info">
+					{ value === 'not_active' && (
+						<button
```

Comment:
IMHO, we can reuse the Button component of `@automattic/components` along with the `emerald-button-style` to implement the busy status quickly.

```suggestion
						<Button
							primary
							busy={ disabled }
```

<img width="548" alt="截圖 2023-04-22 上午12 19 27" src="https://user-images.githubusercontent.com/6869813/233686557-a0fd4962-87b9-45db-b763-10396cb16c2b.png">

Reply:
I don't see a very clean way to use the busy status, and also I don't think we have designs for the colors. I'll defer this to future improvements. Good point tho.